gwdetchar/saturation.py

```diff
@@ -186,7 +186,8 @@ def is_saturated(channel, cache, start=None, end=None, indicator='LIMEN',
             channels[i] = c[:-6]
     # check limit if set
     indicators = ['{}_{}'.format(c, indicator) for c in channels]
-    data = get_data(indicators, start, start+1, source=cache, nproc=nproc)
+    gps = file_segment(cache[0])[0]
```

Comment:
@duncanmmacleod, I think this is missing an import statement for the file_segment utility, otherwise looks good to me.